tests/testw3emc.f

```diff
@@ -0,0 +1,317 @@
+       PROGRAM TESTW3MC
+C
```

Comment:
Move this documentation block to the top of the file (above PROGRAM TESTW3MC) and convert it to Doxygen.

Reply:
I need Kyle to help in cmake to compile this program.

Reply:
@kgerheiser can you assist with getting this test working in CMake?

Reply:
Yes